getting-started-guides/on-prem-cluster/standalone-scala.md

```diff
@@ -66,6 +67,48 @@ ${SPARK_HOME}/sbin/start-slave.sh ${SPARK_MASTER} -c ${SPARK_CORES_PER_WORKER}
 
 Note that in this example the Master and Worker processes are both running on the same host. This is not a requirement, as long as all hosts that are used to run the Spark app have access to the dataset.
 
+<span id="etl">Launch Mortgage ETL</span>
+---------------------------
+If user wants to use a larger size dataset other than the default one, we provide an ETL app to process raw Mortgage data.
+
+Variables required to run spark-submit command:
+```
+# path to xgboost4j_spark/libs
+export LIBS_PATH=/home/xgboost4j_spark/lib
+
+# additional jars for XGBoost4J example, use corresponding cudf jar accoding to your cuda runtime version.
+export SPARK_JARS=${LIBS_PATH}/cudf-0.14-cuda10-2.jar
+
+# Rapids plugin jar, working as the sql plugin on Spark3.0
+export JAR_RAPIDS=${LIBS_PATH}/rapids-4-spark_2.12-0.1.0.jar
+
+# Example jar built via mvn command
+export JAR_EXAMPLE=${LIBS_PATH}/sample_xgboost_apps-0.2.2.jar
+
+```
+
+Run spark-submit
+```
+${SPARK_HOME}/bin/spark-submit \
+    --master spark://$HOSTNAME:7077 \
+    --executor-memory 32G \
+    --jars ${SPARK_JARS},${JAR_RAPIDS}\
```

Comment:
For standalone, seems it will not work using "--jars" here, need to move these jar files to <SPARK_HOME>/jars/

Reply:
Updated. Yes, "--jars" doesn't work on standalone. I've add a step before launching spark standalone cluster. Pls review.